cardano-cli/test/cli/genesis-create/run

```diff
@@ -78,4 +78,32 @@ if test "${keyhash_delegate_count}" != "${gendlg_count}" ; then
     error=1
     fi
 
+# Check
+
+for i in $(seq 1 $gendlg_count)
+do
+    # Check genesis keys:
+    assert_file_exists "${TEST_DIR}/genesis-keys/genesis${i}.skey"
+    assert_file_exists "${TEST_DIR}/genesis-keys/genesis${i}.vkey"
+
+    assert_string 'Genesis signing key' "${TEST_DIR}/genesis-keys/genesis${i}.skey"
+    assert_string 'Genesis verification key' "${TEST_DIR}/genesis-keys/genesis${i}.vkey"
+
+    # Check delegate keys:
+    assert_file_exists "${TEST_DIR}/delegate-keys/delegate${i}.skey"
+    assert_file_exists "${TEST_DIR}/delegate-keys/delegate${i}.vkey"
+    assert_file_exists "${TEST_DIR}/delegate-keys/delegate-opcert${i}.counter"
+
+    assert_string 'Node operator signing key' "${TEST_DIR}/delegate-keys/delegate${i}.skey"
```

Comment:
I guess there is no need to check every `i` file for titles/headers, should be enough for 1 of each kind

Reply:
i am fine with that being tested in a for loop. I would just as the you follow the somewhat idiosyncratic indenation style already present in these  (2 space indent, `done` indented the same as the body and `do` in the same line as the `for` with a semicolon.